contracts/MechMarketplace.sol

```diff
@@ -468,24 +467,24 @@ contract MechMarketplace is IErrorsMarketplace {
         // Decrease the number of undelivered requests
         numUndeliveredRequests--;
         // Increase the amount of requester delivered requests
-        mapDeliveryCounts[requester]++;
+        mapDeliveryCounts[mechDelivery.requester]++;
         // Increase the amount of mech delivery counts
-        mapAgentMechDeliveryCounts[msg.sender]++;
+        mapMechDeliveryCounts[msg.sender]++;
         // Increase the amount of mech service multisig delivered requests
         mapMechServiceDeliveryCounts[mechServiceMultisig]++;
```

Comment:
We still need both - one is for mech statistics, another is for multisig activity tracker.